website/react-magma-docs/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-magma-docs",
   "description": "React Magma docs",
-  "version": "5.0.10",
+  "version": "5.0.11-next.14",
```

Comment:
Not that it really matters but we could argue all the charts changes is a minor change for the docs.